client/elements/sc-site-layout.js

```diff
@@ -346,8 +356,9 @@ class SCSiteLayout extends LitLocalized(LitElement) {
       }
       let displaySettingMenu = store.getState().displaySettingMenu;
       let displaySuttaParallels = store.getState().displaySuttaParallels;
+      let displaySuttaToC = store.getState().displaySuttaToC;
```

Comment:
Why you use let instead of const? You're not changing the reference or value of it. You should use const as much as it possible, stateless code is easier to understand.